buildah-core/src/test/java/io/jshift/buildah/core/commands/BudCommandTest.java

```diff
@@ -29,4 +31,17 @@ public void should_execute_bud_command() {
         assertThat(cliCommand).containsExactly(transform("bud --tls-verify=false --file Dockerfile --file Dockerfile.fed -t targetImg dockerFileContext"));
 
     }
+
+    @Test
+    public void should_add_build_arg() {
+
+        Map<String, String> buildArgs = new HashMap<>();
+        buildArgs.put("TEST", "TEST_VAL");
+        List<String> dockerfileList = new ArrayList<>();
```

Comment:
Should be possible to replace by:
`List<String> dockerfileList = Collections.singletonList("Dockerfile");